src/pages/bag/select-count/select-count.ts

```diff
@@ -0,0 +1,23 @@
+import {Component} from "@angular/core";
+import {NavParams} from "ionic-angular";
+
+@Component({
+  selector: 'page-select-count',
+  templateUrl: 'select-count.html'
+})
+export class SelectCount {
+  count = [];
+
+  constructor(public navParams: NavParams) {
+    const _count = this.navParams.get('count');
+    let _countArr = [];
+    for(let i = 1; i <= _count; i++) {
+      let _countObj = {
+        key: i,
+        value: i
```

Comment:
What is different between key and value when both have i value,
The value property should have the number in Persian format